pages/[id].js

```diff
@@ -0,0 +1,23 @@
+import { useRouter } from "next/router";
+import ActivityDetails from "@/components/ActivityDetails";
+import { dummyData } from "@/lib/dummyData";
+import Link from "next/link";
+
+export default function Activity({}) {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const activity = dummyData.find((activity) => activity.id === id);
+
+  if (!activity) {
+    <p>Activity not found</p>;
+    return;
```

Comment:
This doesn't work. Have you tried? Change the Browser URL to an unexisting id and test it.

Reply:
Thanks, we fixed it (together)